### PR TITLE
Add Phobos 3 Documentation

### PR DESCRIPTION
**Pull request by LightBender**
This adds the documentation from the Phobos VS Design repo into the Phobos repo and adds a Phobos 3 specific README to make it easier to find the documentation.

### CODE REVIEW

File: phobos/PackageMap.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    | exponential
+    | hardware
+    | operations
+    | remainder
+    | rounding
+    | traits
+    | trigonometry
+    | special
+  | meta
+  | numeric
```
Comment (thewilsonator) [inline phobos/PackageMap.md:60]: I would posit that we should move `numeric` into `math` (and split it into `customfloat` and numerical algorithms (root, min, max, dotProduct, divergences etc).

**Review by thewilsonator — APPROVED** [review]
Otherwise looks good

File: phobos/PackageMap.md (hunk 1 above)
Comment (LightBender) [reply phobos/PackageMap.md]: For now I'll just move the package to math. We can decide what we want to do with it in detail when we start merge numeric into Phobos 3.

Comment (thewilsonator) [reply phobos/PackageMap.md]: Sounds good.

**Pull request merged**